Python/optimizer_bytecodes.c

```diff
@@ -1055,6 +1055,10 @@ dummy_func(void) {
         sym_set_const(callable, (PyObject *)&PyUnicode_Type);
     }
 
+    op(_CALL_LEN, (callable[1], self_or_null[1], args[oparg] -- res)) {
```

Comment:
I think you can mark these as `unused`:
```suggestion
    op(_CALL_LEN, (unused, unused, unused -- res)) {
```

Reply:
Let's not do that. I prefer the inst definitions in this file to match `bytecodes.c`. In fact, it was always my intention to validate it, but if you check out the validation function in `Tools/cases_generator/optimizer_generator.py`, it says TODO :wink: . I just had no time to get around to do it.

Reply:
Ah, I didn't know that. I've been using `unused` in my recent PRs.. I'll stop doing that then :)

Reply:
yes, I tend to leave as closer as possible to the `bytecodes.c`

Reply:
> Let's not do that. I prefer the inst definitions in this file to match `bytecodes.c`. In fact, it was always my intention to validate it, but if you check out the validation function in `Tools/cases_generator/optimizer_generator.py`, it says TODO 😉 . I just had no time to get around to do it.

Would this be mostly about ensuring the stack inputs and outputs are identical between `bytecodes` and `optimizer_bytecodes`? I can look into that if you want?

Reply:
Yes please. You'd need to add the code here https://github.com/python/cpython/blob/main/Tools/cases_generator/optimizer_generator.py#L33.

Reply:
on it!